src/vue-components/panel.vue

```diff
@@ -142,7 +142,7 @@
             </div>
             <div v-show="data.tabs.info" class="alpheios-panel__tab-panel alpheios-panel__content_no_top_padding alpheios-panel__tab__info">
                 <div class="alpheios-lookup__panel" v-if="data.infoComponentData">
-                  <lookup :uiController="uiController" :parentLanguage="data.infoComponentData.languageName"></lookup>
+                  <lookup :uiController="uiController" @change="settingChanged"></lookup>
```

Comment:
See my other comments about state. I think the setting values need to respond to changes in page state, which is why I was sending it the parentLanguage